src/sentry.rs

```diff
@@ -0,0 +1,90 @@
+use std::borrow::Cow;
+use tracing_subscriber::prelude::*;
+
+use crate::manifest::manifest_version;
+
+#[must_use]
+#[allow(dead_code)]
+pub struct Guard(
+    Option<tracing_appender::non_blocking::WorkerGuard>,
+    Option<sentry::ClientInitGuard>,
+);
+
+pub fn setup() -> Guard {
+    Guard(tracing_setup(), sentry_setup())
+}
+
+#[inline]
+fn is_whitespace_ascii(b: u8) -> bool {
+    b <= 0x20u8
+}
+
+fn do_not_track() -> bool {
+    if let Some(value) = std::env::var_os("DO_NOT_TRACK") {
+        value
+            .as_encoded_bytes()
+            .iter()
+            .any(|b| !is_whitespace_ascii(*b))
```

Comment:
I'd probably just turn this into `!matches(value.to_string_lossy().to_lowercase().trim(), "0", "false", "no", "")` to catch the falsy values as well

Reply:
way easier to read indeed! thank you!